ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -386,34 +386,59 @@ ScrollView {
                 }
             }
 
-            // TODO: replace with StatusQ component
+            StatusSectionHeadline {
+                text: qsTr("Developer features")
+                topPadding: Style.current.bigPadding
+                bottomPadding: Style.current.padding
+            }
+
+            Separator {
+                anchors.topMargin: Style.current.bigPadding
+                anchors.left: parent.left
+                anchors.leftMargin: -Style.current.padding
+                anchors.right: parent.right
+                anchors.rightMargin: -Style.current.padding
+            }
+
             StatusSettingsLineButton {
-                text: qsTr("Download messages")
-                isSwitch: true
-                switchChecked: localAccountSensitiveSettings.downloadChannelMessagesEnabled
+                text: qsTr("Full developer mode")
+                isBadge: treu
```

Comment:
🤷 
```suggestion
                isBadge: true
```

Reply:
That was a test to know if you really review the code. Congrats you pass ;)

Reply:
Ironically "treu" is the german word for "faithful".

Just leaving this here :D